src/portal-impl.c

```diff
@@ -719,7 +719,10 @@ find_portal_implementation (const char *interface)
 GPtrArray *
 find_all_portal_implementations (const char *interface)
 {
+  const char **desktops;
   GPtrArray *impls;
+  GList *l;
+  int i;
```

Comment:
If you apply the previous suggestions, you'll be able to remove these variables from here

Reply:
I did apply them—in a separate commit, as I updated `find_portal_implementation` to match as well.